.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,35 @@
+name: Bug Report
+description: Report any bugs you encounter.
+body:
+  - type: textarea
+    attributes:
+      label: What is the issue?
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
+        - Effiw
+        - Effis
+        - Eludris
```

Comment:
I don't feel like these scopes make any sense. Effis/Effiw is part of Eludris, Effiw as in effiwbin? That should go to enoki's repo.